benchmark/Cajita_FastFourierTransformPerformance.cpp

```diff
@@ -0,0 +1,232 @@
+/****************************************************************************
+ * Copyright (c) 2018-2021 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "Cabana_BenchmarkUtils.hpp"
+
+#include <Cajita.hpp>
+
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Random.hpp>
+
+#include <fstream>
+#include <iostream>
+#include <random>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include <mpi.h>
+
+using namespace Cajita;
+
+//---------------------------------------------------------------------------//
+// Performance test.
+template <class Device, class HostBackendType>
+void performanceTest( std::ostream& stream, MPI_Comm comm,
+                      const std::string& test_prefix )
+{
+    // Domain size setup
+    std::array<double, 3> global_low_corner = { 0.0, 0.0, 0.0 };
+    std::array<double, 3> global_high_corner = { 1.0, 1.0, 1.0 };
+    std::array<bool, 3> is_dim_periodic = { true, true, true };
+
+    // Declare the grid size per dimension
+    // currently, testing 3dims+symmetric
+    std::vector<double> grid_sizes_per_dim = { 10, 200 };
```

Comment:
Let's modify to match the other benchmarks with a "small" (CI/default) test and "large" (Summit). I think that was changed after you started this test. Check the Actions output to make sure the CI run is ~1  sec

Reply:
@streeve thanks! I've changed the FFT sizes, since the old default was taking ~13 seconds.
In the last CI with new defaults, Jenkins ran the FFT performance test in 0.77s for HIP, and 2.26s for CUDA.